File: Python/1045 - Triangle Types.py
a,b,c=map(float,input().split())
if a>=(b+c):
  print("NAO FORMA TRIANGULO")
elif  b>=a+c:
     print("NAO FORMA TRIANGULO")
elif c>=b+a:
     print("NAO FORMA TRIANGULO")
else:
    if a**2 == (b**2 + c**2):
      print("TRIANGULO RETANGULO")
    elif b**2 == (a**2 + c**2):
        print("TRIANGULO RETANGULO")
    elif c**2 == (a**2 + b**2):
        print("TRIANGULO RETANGULO")
    if (a**2) > (b**2 + c**2):
        print("TRIANGULO OBTUSANGULO")
    elif (b**2) > (a**2 + c**2):
        print("TRIANGULO OBTUSANGULO")
    elif (c**2) > (a**2 + b**2):
        print("TRIANGULO OBTUSANGULO")
    if (a**2) < (b**2 + c**2):
      if (b**2) < (a**2 + c**2):
        if (c**2) < (a**2 + b**2):
           print("TRIANGULO ACUTANGULO")
    if a==b==c:
        print("TRIANGULO EQUILATERO")
    if a == b != c:
        print("TRIANGULO ISOSCELES")
    elif a == c != b:
        print("TRIANGULO ISOSCELES")
    elif b == c != a: 
        print("TRIANGULO ISOSCELES")

File: Java/1045 - Triangle Types.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();

        if (a >= (b + c) || b >= (a + c) || c >= (a + b)) {
            System.out.println("NAO FORMA TRIANGULO");
        } else {
            if (a * a == (b * b + c * c) || b * b == (a * a + c * c) || c * c == (a * a + b * b)) {
                System.out.println("TRIANGULO RETANGULO");
            } else if (a * a > (b * b + c * c) || b * b > (a * a + c * c) || c * c > (a * a + b * b)) {
                System.out.println("TRIANGULO OBTUSANGULO");
            } else if (a * a < (b * b + c * c) && b * b < (a * a + c * c) && c * c < (a * a + b * b)) {
                System.out.println("TRIANGULO ACUTANGULO");
            }

            if (a == b && b == c) {
                System.out.println("TRIANGULO EQUILATERO");
            } else if (a == b && b != c || a != b && b == c || a == c && c != b) {
                System.out.println("TRIANGULO ISOSCELES");
            }
        }

        scanner.close();
    }
}